source/ui/beamline/AMExtendedControlEditor.cpp

```diff
@@ -456,6 +462,20 @@ QSize AMExtendedControlEditor::sizeHint() const{
 	return newHint;
 }
 
+void AMExtendedControlEditor::onControlMinimumValueChanged()
+{
+	if (control_) {
```

Comment:
This seems like a fundamental limitation then because if you pass a control to a control editor after it has connected you'd be left in limbo.  Wouldn't having the means of checking be a more robust solution?